sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/AggregateDocumentQueryExecutionContext.java

```diff
@@ -114,18 +103,49 @@ public AggregateDocumentQueryExecutionContext (IDocumentQueryExecutionComponent<
                 }).flux();
     }
 
-    public static <T extends Resource>  Flux<IDocumentQueryExecutionComponent<T>> createAsync(
-            Function<String, Flux<IDocumentQueryExecutionComponent<T>>> createSourceComponentFunction,
-            Collection<AggregateOperator> aggregates,
-            String continuationToken) {
+    public static <T extends Resource> Flux<IDocumentQueryExecutionComponent<T>> createAsync(
+        Function<String, Flux<IDocumentQueryExecutionComponent<T>>> createSourceComponentFunction,
+        Collection<AggregateOperator> aggregates,
+        Map<String, AggregateOperator> groupByAliasToAggregateType,
+        List<String> groupByAliases,
+        boolean hasSelectValue,
+        String continuationToken) {
 
         return createSourceComponentFunction
-                .apply(continuationToken)
-                .map( component -> { return new AggregateDocumentQueryExecutionContext<T>(component, aggregates);});
+                   .apply(continuationToken)
+                   .map(component -> {
+                       return new AggregateDocumentQueryExecutionContext<T>(component,
```

Comment:
nit: return can be removed , implicit lambda

Reply:
Wrote it like this Just to be consistent with other QueryExecutionContexts. Will eventually clean up all of them together